MAINTAINERS.md

```diff
@@ -0,0 +1,42 @@
+# Maintainers
```

Comment:
While you are here, do you want to add the other repos: SR-IOV DP & CNI and device-info-spec?

Reply:
Good call! Thanks.